ProcessLib/PhaseField/PhaseFieldFEM.h

```diff
@@ -46,7 +46,7 @@ struct IntegrationPointData final
 
     typename BMatricesType::KelvinVectorType sigma_tensile, sigma_compressive,
         sigma_real_prev, sigma_real;
-    double strain_energy_tensile;
+    double strain_energy_tensile = 0.0;
```

Comment:
Because none of the members is initialized here, I'd recommend to move the initializations together.

Reply:
It has been moved.